crates/bevy_ecs/src/lib.rs

```diff
@@ -2768,3 +2771,10 @@ mod tests {
         fn custom_clone(_source: &SourceComponent, _ctx: &mut ComponentCloneCtx) {}
     }
 }
+
+/// Event triggered when a hotpatch happens.
```

Comment:
```suggestion
/// Event sent when a hotpatch happens.
```
I think we should reserve the term "triggered" for triggers